common/flatpak-transaction.c

```diff
@@ -2431,6 +2444,82 @@ resolve_all_ops (FlatpakTransaction *self,
   return TRUE;
 }
 
+
+static gboolean
+request_tokens_for_remote (FlatpakTransaction *self,
+                           const char         *remote,
+                           GList              *ops,
+                           GCancellable       *cancellable,
+                           GError            **error)
+{
+  g_autoptr(GString) refs_as_str = g_string_new ("");
+  GList *l;
+  const char *token;
+
+  for (l = ops; l != NULL; l = l->next)
+    {
+      FlatpakTransactionOperation *op = l->data;
+
+      if (l != ops)
+        g_string_append (refs_as_str, ", ");
+      g_string_append (refs_as_str, op->ref);
+    }
+
+  g_debug ("Requesting tokens for remote %s, refs: %s", remote, refs_as_str->str);
+
+  token = g_getenv ("FLATPAK_TEST_TOKEN");
```

Comment:
Nitpick: Maybe put a `FIXME` comment in here for now to make it clear that this is temporary. (Mostly for the poor git historians who have to dig through this in future.)

Reply:
I expect to rewrite this branch a fair bit before landing, but sure, that would be nice.